src/control/run_go_tests.sh

```diff
@@ -2,6 +2,57 @@
 ## Run linters across control plane code and execute Go tests
 set -eu
 
+function get_cpu_perf()
+{
+	test_file=$(mktemp primebenchXXX.go)
+
+	cat <<EndOfScript>$test_file
+package main
+
+import (
+	"fmt"
+	"math"
+	"time"
+)
+
+func main() {
+	primeCh := make(chan int)
+	primes := 0
+
+	go func() {
+		isPrime := func(v int) bool {
+			for i := 2; i <= int(math.Floor(math.Sqrt(float64(v)))); i++ {
+				if v%i == 0 {
+					return false
+				}
+			}
+			return v > 1
+		}
+		for i := 2; ; i++ {
+			if isPrime(i) {
+				primeCh <- i
+			}
+		}
+	}()
+
+	timeout := 1 * time.Second
+	timer := time.NewTimer(timeout)
+	for {
+		select {
+		case <-timer.C:
+			fmt.Printf("%d primes in %s", primes, timeout)
+			return
+		case <-primeCh:
+			primes++
+		}
+	}
+}
+EndOfScript
+
+	echo $(go run $test_file)
```

Comment:
(lint) Quote this to prevent word splitting. [SC2046]
(lint) Useless echo? Instead of 'echo $(cmd)', just use 'cmd'. [SC2005]
(lint) Double quote to prevent globbing and word splitting. [SC2086]